mp/src/game/shared/baseplayer_shared.cpp

```diff
@@ -1726,6 +1726,9 @@ void CBasePlayer::CalcVehicleView(
 void CBasePlayer::CalcObserverView( Vector& eyeOrigin, QAngle& eyeAngles, float& fov )
 {
 #if defined( CLIENT_DLL )
+#ifdef NEO
+	m_Local.m_iHideHUD &= ~HIDEHUD_CROSSHAIR;
+#endif
```

Comment:
This has the side effect of displaying the crosshair to spectators upon joining  the server:

![20240625201225_1](https://github.com/NeotokyoRebuild/neo/assets/6595066/2dd9a0c2-6094-4b4f-a68d-13fc1cc66573)

Do we need to clear this bit here for some purpose?

Reply:
Whoops that's a screw up, should be the otherway round.